src/components/Menu/entries.js

```diff
@@ -53,7 +55,23 @@ export const ReadonlyEntries = [{
 	},
 }]
 
-export default [
+export const ReadOnlyEditEntries = [{
+	key: 'edit',
+	label: t('text', 'Edit'),
+	forceLabel: true,
+	icon: Pencil,
+	click: ({ $readOnlyActions }) => $readOnlyActions.toggle(),
+}]
+
+export const ReadOnlyDoneEntries = [{
+	key: 'done',
+	label: t('text', 'Done'),
+	keyChar: 'esc',
```

Comment:
This does not seem to work, would drop it for now.

Reply:
@max-nextcloud I tried to push a commit for this but failed. Maybe you can?

Reply:
fyi 
```
git push git@github.com:benthie/text.git feat/read-only-setting 
```
did the trick for me.